sources/osg/State.js

```diff
@@ -821,15 +810,15 @@ MACROUTILS.createPrototypeObject(
             if (_attributeArray[index] === undefined) {
                 this._createAttributeStack(_attributeArray, index, attribute);
             } else {
-                _attributeArray[index].globalDefault = attribute;
+                _attributeArray[index]._globalDefault = attribute;
```

Comment:
you renamed one var upper in the code _attributeArray to attributeArray . If you change one change them all, be consistent